framework/include/auxkernels/CopyValueAux.h

```diff
@@ -24,8 +24,13 @@ class CopyValueAux : public AuxKernel
 protected:
   virtual Real computeValue() override;
 
+  // variable used to specify state being copied
+  unsigned short _state;
+
   /// The variable to project from
   const VariableValue & _v;
```

Comment:
Get rid of all the other VariableValues now. With the change I sent in the .C file you don't need to store all of these anymore.